python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -14,6 +15,18 @@
 from dask_cudf.groupby import AGGS, CUMULATIVE_AGGS, _aggs_supported
 
 
+def check_groupby_result(ddf):
```

Comment:
This doesn't really check any results, perhaps `assert_has_hlg_layers` and name the layers?

```
def assert_has_hlg_layers(ddf, *prefixes):
    for prefix in prefixes:
        try:
            layer = hlg_layer(ddf.dask, prefix)
        except KeyError:
            raise AssertionError(f"Dask dataframe does not contain dask-cudf groupby layer with prefix {prefix}")
```

And then

```
assert_has_hlg_layers(ddf, "cudf-aggregate-chunk", "cudf-aggregate-agg")
```